README.md

```diff
@@ -20,15 +20,27 @@ Questions? Comments? Start by perusing our [GitHub discussions](https://github.c
 
 ### Screenshots
 
-![Screenshot of main page](https://raw.githubusercontent.com/nautobot/nautobot/develop/nautobot/docs/media/screenshot1.png "Main page")
+![Screenshot of main page](nautobot/docs/media/screenshot1.png "Main page")
 
 ---
 
-![Screenshot of rack elevation](https://raw.githubusercontent.com/nautobot/nautobot/develop/nautobot/docs/media/screenshot2.png "Rack elevation")
+![Screenshot of config contexts](nautobot/docs/media/screenshot2.png "Config Contexts")
 
 ---
 
-![Screenshot of prefix hierarchy](https://raw.githubusercontent.com/nautobot/nautobot/develop/nautobot/docs/media/screenshot3.png "Prefix hierarchy")
+![Screenshot of prefix hierarchy](nautobot/docs/media/screenshot3.png "Prefix hierarchy")
+
+---
+
+![Screenshot of golden config](nautobot/docs/media/screenshot4.png "Golden config")
```

Comment:
Not sure it's ideal to put screenshots of plugins in the main repo's README... maybe under a separate subheading `#### Plugin Screenshots`? Not sure.

Reply:
I think that could work too.  I was trying to think of a way to showcase it's extensibility, but separating them out into a separate section might be a better idea.

Reply:
Updated into a separate section